financialaid/backend_tests.py

```diff
@@ -0,0 +1,91 @@
+"""
+Tests for financialaid view
+"""
+from django.db.models.signals import post_save
+
+from factory.django import mute_signals
+
+from courses.factories import ProgramFactory
+from dashboard.models import ProgramEnrollment
+from financialaid.backend import determine_tier_program, determine_auto_approval
+from financialaid.factories import TierProgramFactory, FinancialAidFactory
+from profiles.factories import ProfileFactory
+from search.base import ESTestCase
+
+
+class FinancialAidBackendTests(ESTestCase):
+    """
+    Tests for financialaid views
+    """
+    def setUp(self):
+        """
+        Sets up user, logs user in, and enrolls user in program
+        """
+        with mute_signals(post_save):
+            self.profile = ProfileFactory.create()
+        self.program = ProgramFactory.create(
+            financial_aid_availability=True
+        )
+        self.tiers = {
+            "0k": TierProgramFactory.create(program=self.program, income_threshold=0, current=True),
+            "15k": TierProgramFactory.create(program=self.program, income_threshold=15000, current=True),
+            "50k": TierProgramFactory.create(program=self.program, income_threshold=50000, current=True),
+            "100k": TierProgramFactory.create(program=self.program, income_threshold=100000, current=True)
+        }
+        self.program_enrollment = ProgramEnrollment.objects.create(
+            user=self.profile.user,
+            program=self.program
+        )
```

Comment:
If all this code does not need to run once per test, please move it to `setUpTestData`